frontend/src/lib/components/UniversalSearch/UniversalSearchPopover.tsx

```diff
@@ -32,7 +32,7 @@ export interface UniversalSearchPopoverProps<ValueType = TaxonomicFilterValue>
     groupType: TaxonomicFilterGroupType
     value?: ValueType
     onChange?: (value: ValueType, groupType: TaxonomicFilterGroupType, item: SearchDefinitionTypes) => void
-    groupTypes?: TaxonomicFilterGroupType[]
+    groupTypesArray?: TaxonomicFilterGroupType[]
```

Comment:
@tomasfarias I suspect the type error exists because we renamed it here too 😅 .

I think there are only 3-4 places that actually use groupTypes for Group Analytics Group types; rest are all TaxonomicFilter related, we can revert all of these

Reply:
one by one I'm afraid, buuut that's what the rename helps with, if you just change `groupsModel`, and then run `pnpm typescript:check` it will show you all places that referenced it which you now need to change; and don't need to bother about the rest.

So revert all but the `groupsModel` file, see where ts errors pop up, rename those places, confirm they're not accessing by index, and we're done 👍

Reply:
Ok, I've renamed all the files that popped up in `pnpm typescript:check`. Thank you!

Reply:
Nevermind, now tests are failing...